triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -63,12 +63,18 @@ final case class CompletionMsg(c: Completion) extends TriggerMsg
 final case class TransactionMsg(t: Transaction) extends TriggerMsg
 final case class HeartbeatMsg() extends TriggerMsg
 
-final case class TypedExpr(expr: Expr, ty: TypeConApp)
+final case class TriggerDefinition(
+    id: Identifier,
+    ty: TypeConApp,
+    version: Trigger.Version,
+    lowLevel: Trigger.Level,
```

Comment:
Maybe `level` would be a better name than `lowLevel`?

Reply:
good catch !